docs/GettingStarted/01-Why-Fedimint.md

```diff
@@ -1,75 +1,78 @@
+Fedimint aims to address this by distributing the custodianship across millions of communities, making it simple for communities to bank themselves. Fedimint allows Bitcoiners to onboard newCoiners themselves, assisting them in their custody and payment model. Instead of referring a newCoiner to a third party custodian, you can onboard them yourself as part of a Federation.
 
-Put another way it allows you to **be your mum's / friends / villages bank**, so they don't have to. 
+Put another way it allows you to **be your mum's / friends / villages bank**. We call these close, trusted relationships "2nd party custodians". Fedimint federation guardians should be close friends and family members that you know personally, and can directly influence should they ever attempt to violate your trust.
 
-This provides bitcoiners with a third option between 3rd party centralized custodians and self custody as shown in the figure below. 
+This provides bitcoiners with a third option between 3rd party centralized custodians and self custody as shown in the figure below.
 
 ![The spectrum and trade offs for Fedimint Custody](/img/raw-figures/fm-spectrum-custody.excalidraw.png)
 
-Fedimint has also been design to interoperate with the bitcoin lightning network. 
+Most importantly, Fedimint is directly interoperable with the Lightning Network. Any Fedimint user can at any time move their funds into their own lightning wallet or onto their own node.
 
 This allows Fedimint users to remain part of the wider lightning network merchants, making instant payments and moving between Fedimints without managing additional complexity.
 
-As such as a user who previously may have used 3rd party custody for convenience can retain that convenience whilst improving their privacy and control of their finances. 
+As such as a user who previously may have used 3rd party custody for convenience can retain that convenience whilst improving their privacy and control of their finances.
 
 :::note
-There is a [trade off to be considered](../CommonTerms/CustodialRisk) here as you are trusting a federation with your bitcoin custody.  As such it will be important to "know your federation".
+There is a [trade off ](../CommonTerms/CustodialRisk) here as you are trusting a federation with your bitcoin custody. As such it will be important to "know your federation".
 :::
 
 ## Financial Privacy
-By implementing [Blind Signatures and Chaumian eCash protocols](/docs/CommonTerms/Blind%20Signatures) inside the Fedimint, the identifying details of the fm-BTC eCash tokens are unknowable by the guardians. 
+
+Fedimint uses [Chaumain e-cash tokens and blinded signatures](/docs/CommonTerms/Blind%20Signatures) to achieve perfect privacy for federation members. Federation guardians cannot correlate inputs and outputs of federation members' transactions, and cannot see the holdings of any individual federation member.
```

Comment:
I get the perfect privacy being a point of eCash schemes, but personally I wouldn't want to oversell that as I think there's reasonable scope to fuck this up with things like the back up process? 

```suggestion
Fedimint uses [Chaumain e-cash tokens and blinded signatures](/docs/CommonTerms/Blind%20Signatures) to achieve privacy for federation members. Federation guardians cannot correlate inputs and outputs of federation members' transactions, and cannot see the holdings of any individual federation member.
```